### PR TITLE
Post: update migrate-from-helm-v2-to-helm-v3 with clean up feature

### PR DESCRIPTION
**Pull request by rimusz**
Signed-off-by: rimas <rmocius@gmail.com>

### CODE REVIEW

File: content/blog/2019-09-11-migrate-from-helm-v2-to-helm-v3.md [hunk 1]
```diff
@@ -213,4 +215,65 @@ $ helm3 2to3 convert postgres --tiller-out-cluster
...
+
+When you are ready to clean up Hem v2 data, just run that command without `--dry-run` flag.
+
+**NOTE:** The `cleanup` command will remove the Helm v2 Configuration, Release Data and Tiller Deployment. It cleans up all releases managed by Helm v2. It will not be possible to restore them if you haven't made a backup of the releases. Helm v2 will not be usable afterwards.
+
+If you are using Tillerless Helm v2, just add `--tiller-out-cluster` to clean up Helm v2 data.
+
+The plugin also supports non default Helm v2 `home` data folder and Tiller releases `namespace`:
+
+```
```
Comment (hickeyma) [inline content/blog/2019-09-11-migrate-from-helm-v2-to-helm-v3.md]: Add : $ export HELM_V2_HOME=$PWD/.helm2

File: content/blog/2019-09-11-migrate-from-helm-v2-to-helm-v3.md [hunk 2]
```diff
@@ -38,25 +38,26 @@ Let's install it:
...
+Downloading and installing helm-2to3 v0.1.2 ...
+https://github.com/helm/helm-2to3/releases/download/v0.1.0/helm-2to3_0.1.2_darwin_amd64.tar.gz
 Installed plugin: 2to3
 ```
 
 ```
 $ helm3 plugin list
 NAME	VERSION	DESCRIPTION
-2to3	0.1.0  	migrate Helm v2 configuration and releases in-place to Helm v3
+2to3	0.1.2  	migrate Helm v2 configuration and releases in-place to Helm v3
```
Comment (hickeyma) [inline content/blog/2019-09-11-migrate-from-helm-v2-to-helm-v3.md]: migrate Helm v2 configuration and releases in-place to Helm v3 --> migrate and clean up Helm v2 configuration and releases in-place to Helm v3

We can update the plugin description in the code base subsequently

File: content/blog/2019-09-11-migrate-from-helm-v2-to-helm-v3.md (hunk 1 above)
Comment (rimusz) [reply content/blog/2019-09-11-migrate-from-helm-v2-to-helm-v3.md]: done already in https://github.com/helm/helm-www/pull/285/commits/f84d5c422066ff9f1e960d31c92bc2637d76b4a4

File: content/blog/2019-09-11-migrate-from-helm-v2-to-helm-v3.md (hunk 2 above)
Comment (rimusz) [reply content/blog/2019-09-11-migrate-from-helm-v2-to-helm-v3.md]: done

**Comment by rimusz** [comment]
thanks

**Pull request merged**